src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderIdentificationAlgorithm.cpp

```diff
@@ -915,6 +930,8 @@ namespace OpenMS
       transition.setLibraryIntensity(iso_it->getIntensity());
       transition.setMetaValue("annotation", annotation);
       transition.setPeptideRef(peptide_id);
+
+      //TODO what about transition charge? A lot of DIA scores depend on it and default to charge 1 otherwise.
```

Comment:
TODO remove. This comment should be solved by using the MS1 isotope scores only. I think "per-transition" isotopes do not make much sense here. They will overlap and the negative pre-isotopes during scoring may hit previous transitions in our case.

Reply:
But I might need to check the other MS2 scores if they depend on a transition charge.